git TestUpdateStore is flaky

```
=== FAIL: internal/storage/git TestUpdateStore/move_policy_out_of_policy_dir (0.00s)
    store_test.go:367: 
        	Error Trace:	store_test.go:367
make: *** [Makefile:93: test-integration] Error 1
        	Error:      	Received unexpected error:
        	            	failed to do work: rename /tmp/TestUpdateStore161186429/001/source/policies/git_leave_request_005.default.yaml /tmp/TestUpdateStore161186429/001/source/ignore/git_leave_request_005.default.yaml: no such file or directory
        	Test:       	TestUpdateStore/move_policy_out_of_policy_dir
    --- FAIL: TestUpdateStore/move_policy_out_of_policy_dir (0.00s)
```

--- a/internal/storage/git/store_test.go
+++ b/internal/storage/git/store_test.go
@@ -119,7 +119,8 @@ func TestUpdateStore(t *testing.T) {
 
 	rng := rand.New(rand.NewSource(time.Now().Unix())) //nolint:gosec
 	numPolicySets := 20
-	deletedFilesetNumber := 0
+	var deletedFilesetNumber int
+	var renamedFilesetNumber int
 
 	_ = createGitRepo(t, sourceGitDir, numPolicySets)
 
@@ -303,14 +304,13 @@ func TestUpdateStore(t *testing.T) {
 		}, nil)
 
 		checkEvents := storage.TestSubscription(store)
-		moveFilesetNumber := rng.Intn(numPolicySets)
 		for {
-			if moveFilesetNumber != deletedFilesetNumber {
+			renamedFilesetNumber = rng.Intn(numPolicySets)
+			if renamedFilesetNumber != deletedFilesetNumber {
 				break
 			}
-			moveFilesetNumber = rng.Intn(numPolicySets)
 		}
-		pset := genPolicySet(moveFilesetNumber)
+		pset := genPolicySet(renamedFilesetNumber)
 
 		require.NoError(t, commitToGitRepo(sourceGitDir, "Rename policy", func(wt *git.Worktree) error {
 			for file := range pset {
@@ -355,12 +355,12 @@ func TestUpdateStore(t *testing.T) {
 		}, nil)
 
 		checkEvents := storage.TestSubscription(store)
-		moveFilesetNumber := rng.Intn(numPolicySets)
+		var moveFilesetNumber int
 		for {
-			if moveFilesetNumber != deletedFilesetNumber {
+			moveFilesetNumber = rng.Intn(numPolicySets)
+			if moveFilesetNumber != deletedFilesetNumber && moveFilesetNumber != renamedFilesetNumber {
 				break
 			}
-			moveFilesetNumber = rng.Intn(numPolicySets)
 		}
 		pset := genPolicySet(moveFilesetNumber)
 

